indico/modules/events/registration/controllers/display.py

```diff
@@ -258,8 +258,12 @@ def _checkParams(self, params):
     def _process(self):
         form = make_registration_form(self.regform)()
         if form.validate_on_submit():
-            registration = create_registration(self.regform, form.data, self.invitation)
-            return redirect(url_for('.display_regform', registration.locator.registrant))
+            if self.regform.limit_reached:
+                flash(_("Your registration wasn't successful because registrations limit has been reached"), 'error')
```

Comment:
"was not" (the "*n't" forms are quite colloquial)

But I'd go for something like "The maximum number of registrations has already been reached" - with the 'error' style it should still be clear that you did not successfully register.